### ISSUE TITLE
differencial pins?

### ISSUE THREAD

**Issue by fpgasdr** [maribu/at86rf215-breakout#1]
Hi,

the digital differential ports for data(TX and RX) and clock are not propagated so with this board is not possible to use it like with the [ATREB215-XPRO](https://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42398-ATREB215-XPRO-Extension-Board_UserGuide.pdf) eval board.

you plan to add this?.

**Comment by maribu** [comment]
No, but feel free to open a pull request doing so, or to create a fork with this added.

**Comment by eljuligallego** [comment]
OK, I can try. 

any board/connector in mind?. 

My idea is to connect a FPGA board compatible with lvds at this frequencies. something cheap and powerfull like the [EBAZ4205](https://github.com/xjtuecho/EBAZ4205).
